packs/elastic/pack.yaml

```diff
@@ -0,0 +1,10 @@
+---
+name : elastic
```

Comment:
> I think "elasticsearch" ...
That's right. Though we can use this pack for other elastic products like logstash and kibana. What do you think?